src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -69,6 +70,13 @@ public void testBackEndActions() throws Exception {
 
     public void testContent() throws Exception {
 
+        ______TS("Typical case: large session with no sections");
+
+        resultsPage = loginToInstructorFeedbackResultsPageWithViewType("CFResultsUiT.instr",
+                "Session with no sections", true, "question");
+        AssertHelper.assertContains(Const.StatusMessages.FEEDBACK_RESULTS_QUESTIONVIEWWARNING,
+                resultsPage.getStatus());
```

Comment:
Is the status message subjected to variation? If the status message is exact, use `resultsPage.verifyStatus(...)` instead.
If it is, then align `[r]esultsPage.getStatus()` with `[C]onst.StatusMessage`.